content/doc/book/managing/plugins.adoc

```diff
@@ -225,6 +225,67 @@ The configuration(s) created by the disabled plugin behave as if the plugin
 were uninstalled, insofar that they result in warnings on boot but are
 otherwise ignored.
 
+==== Using the Jenkins CLI
+
+It is also possible to enable and disable plugins using the <<cli#,Jenkins CLI>>.
+Since Jenkins 2.136, administrators may invoke the `enable-plugin` command to 
+re-enable again a collection of plugins which were previously disabled and since
+Jenkins 2.151, the `disable-plugin` command is included, which provides a command
+to disable installed plugins.
+
+The `enable-plugin` command receives as parameter a list of plugins to be enabled.
```

Comment:
I think I would remove as parameter, it's more an implementation detail.

Reply:
Removing here and in disable-plugin